### DISCUSSION TITLE
remark cli configuration

### DISCUSSION THREAD

**Discussion by muravjev** [remarkjs/remark#1063] [Q&A] [answered]
I have a github bug-report.md. With following frontmatter(?)
```
---
name: Bug report
about: Create a report to help us improve
---
``` 
and after run `pnpm remark -o -q` the output as follows:
```
***

name: Bug report
about: Create a report to help us improve
-------------
```

**How I can prevent these changes?**

I use `remark-preset-prettier`
And use remark's `remark-toc` to generate toc.

My `.remarkrc.mjs` is
```js
import remarkPrettier from 'remark-preset-prettier';
import remarkToc from 'remark-toc';

const remarkConfig = {
    plugins: [
        [remarkToc, { heading: 'contents', tight: true }],
        remarkPrettier
    ]
};

export default remarkConfig;
```

**Comment by ChristianMurphy** [comment, marked as answer]
Frontmatter is not a standard markdown feature.
And as such, it needs a plugin to enable support.
Add https://github.com/remarkjs/remark-frontmatter to your remark configuration.

**Reply by muravjev** [reply]
Thank you @ChristianMurphy! It works! 👍
